odbc_statement.c

```diff
@@ -2946,6 +2947,18 @@ make_param_array (ODBC_STATEMENT * stmt,
       else
 	{
 	  param_array->ind_array[i] = 0;
+#ifdef CUBRID_ODBC_UNICODE
+	  if (desc_info->type == SQL_WCHAR)
+	    {
+	      void *cci_value;
+
+	      cci_value = odbc_value_to_cci (value_ptr, -8, desc_info->length, 0, 0);
```

Comment:
I think SQL_WCHAR is better than '-8'.

Reply:
Thank you for your comment, I made a mistake while migrating codes from the test version.